addons/backpacks/config.cpp

```diff
@@ -35,6 +35,7 @@ class CfgPatches {
         "A3_Modules_F",
         "A3_UI_F",
         "A3_Structures_F_Items_Electronics",
+        "A3_Supplies_F_Enoch_Bags",
```

Comment:
This whole stuff really isn't needed. tfar_core requires cba_common which requires all of vanilla Arma.
But no need to really fix it either.